src/components/Banner/BannerContextManager.js

```diff
@@ -10,12 +10,23 @@ import React, {useState, useLayoutEffect} from 'react';
 import BannerContext from './BannerContext';
 
 let activeBanner = null;
+let snooze_start_date = null;
```

Comment:
Why is the snake case used here? Shouldn't we follow camel case naming everywhere?